syntax = "proto3";

import "bob_emploi/frontend/api/options.proto";

package bayes.bob.test;


message ScoringModels {
  repeated string filters = 1 [(string_format) = SCORING_MODEL_ID];
}

message UrlLink {
    string link = 1 [(string_format) = URL_FORMAT];
}

message Template {
  string text_template = 1 [(string_format) = SCORING_PROJECT_TEMPLATE];
}
